backend/src/controllers/reports/Tab_34_rpt_PA_StatusDashboard.js

```diff
@@ -21,10 +21,19 @@ controller.Tab_34_rpt_PA_StatusDashboard = async (request, reply) => {
   controller.userRequires(request, what, "reports_read_all");
   try {
     // Get the data from the database.
-    const getDate = async () => new Date();
+    const portfolios = request.query.portfolio;
+    const dashboardResults = await model.Tab_34_rpt_PA_StatusDashboard(portfolios);
```

Comment:
@ASpiteri-BCGov / @robrien-bcgov  wondering if the export name has to be this... 
 model.Tab_34_rpt_PA_StatusDashboard as this is already part of the import, would it be better just to have something that said model.dashboard or model.dashboardResults.  (observation)